### ISSUE TITLE
Gtk 3.20: Fix the massive breakages

### ISSUE THREAD

**Issue by xyl0n** [xyl0n/iris#121]
Gtk 3.20 is moving to some new node system which apparently breaks every single gtk theme ever. In the long run it's better, but is very inconvenient right now.

I expect to fix this sometime next year.

**Comment by coleifer** [comment]
how she lookin?

**Comment by xyl0n** [comment]
The schedule I have right now (super important exams) means that I'm holding this off until end of June (i.e. when exams finish). At that point, it should be a week or two until it's fixed for 3.20: if I choose to rewrite in SASS it may be potentially be a few more weeks but right now I don't see a benefit big enough to be worth the rewrite.

**Comment by onespaceman** [comment]
Have you considered rebasing iris on an updated version of Numix? It will take the pain out of trying to make changes for GTK 3.20, and it's already ported to SASS.

I've started a bit in [this fork](https://github.com/onespaceman/iris-reborn) and it was simple enough to get the basic iris look down. (though I suggest starting from a fresh fork, my changes are a bit crummy and inconsistent)

Edit: [It's actually starting to look pretty good.](https://i.imgur.com/kHgkt4l.png)

**Comment by coleifer** [comment]
Nice work @onespaceman - that's a good approach.

Good luck on your exams @xyl0n!

**Comment by xyl0n** [comment]
@onespaceman nice work, that will probably be the best way to go about it, as long as the credits and licencing is done properly :)
@coleifer thanks, only 7 weeks left but they're intensive. There's like a 10 week holiday afterwards though :D

**Comment by xyl0n** [comment]
@onespaceman I haven't tested it out yet, my machine is still on 3.18 because I'm updating a few things for the other themes so they work on the latest Ubuntu (which uses 3.18) etc. But I'll see how it works out

**Comment by Anachron** [comment]
Hey @xyl0n have you made any progress regarding this?

**Comment by xyl0n** [comment]
@Anachron yep, I've begun a port to SASS for Gtk 3.20, and I'm trying to get it to a state where it resembles the theme again and I can push it to the repo.
